battle-service/src/main/java/com/company/battle/repositories/SettingsRepository.java

```diff
@@ -4,6 +4,8 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.UUID;
+
 @Repository
-public interface SettingsRepository extends JpaRepository<SettingsEntity, Integer> {
+public interface SettingsRepository extends JpaRepository<SettingsEntity, UUID> {
```

Comment:
I believe we talked about making setting embedded object of Game, not separate entity